config/locales/js-en.yml

```diff
@@ -82,6 +83,7 @@ en:
     label_collapse: "Collapse"
     label_collapsed: "collapsed"
     label_collapse_all: "Collapse all"
+    label_committed_at: "committed revision %{revision}"
```

Comment:
best of both worlds proposal by @ulferts (I am writing what he explained):

````yml
label_my_full_sentence: "%{committed_revision_link} at %{date}"
a_placeholder_for_my_full_sentence: "committed revision %{revision}"
````

and then you build your matroska (ruby example):

````ruby
sentence = I18n.t('label_my_full_sentence',
                  committed_revision_link: I18n.t('a_placeholder_for_my_full_sentence',
                    revision: my_revision),
                  date: my_date)
````